app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -50,9 +50,9 @@ class ChooseStep extends React.Component {
   };
 
   renderOption = (downloadType, field) => {
-    const { selectedFields, onFieldSelect } = this.props;
+    const { selectedFields, onFieldSelect, fieldOptions } = this.props;
     const selectedField = get([downloadType.type, field.type], selectedFields);
-    let dropdownOptions;
+    let dropdownOptions, loadingOptions;
```

Comment:
nit: initializing `loadingOptions` to false hints about the type of variable. Good for readability.